mindsdb/integrations/handlers/mysql_handler/Manual_QA.md

```diff
@@ -2,38 +2,66 @@
 
 > **Please submit your PR in the following format after the underline below `Results` section. Don't forget to add an underline after adding your changes i.e., at the end of your `Results` section.**
 
-## Testing MySQL Handler with [Dataset Name](URL to the Dataset)
+## Testing MySQL Handler with [Dry Bean Dataset](https://www.kaggle.com/datasets/muratkokludataset/dry-bean-dataset)
 
 **1. Testing CREATE DATABASE**
 
-```
-COMMAND THAT YOU RAN TO CREATE DATABASE.
+```sql
+CREATE DATABASE mysql_test                  --- datasource connection name
+WITH ENGINE = 'mysql',                      --- database engine
+PARAMETERS = {
+    "host": "cloud.mindsdb.com",            --- host name can be an IP address, or URL
```

Comment:
I see you are still using `"host": "cloud.mindsdb.com"`, and thus, connecting MindsDB to itself.

The idea is that you have your own MySQL database (here is how you can install MySQL database: https://dev.mysql.com/doc/mysql-getting-started/en/#mysql-getting-started-installing).

And then you connect this database to MindsDB:
```
CREATE DATABASE mysql_test                  --- datasource connection name
WITH ENGINE = 'mysql',                      --- database engine
PARAMETERS = {
    "host": "localhost",            --- host name can be an IP address, or URL
    "port": 3306,                           --- port used to make TCP/IP connection
    "database": "dry_bean_db",              --- database name
    "user": "demo-user",                    --- database user
    "password": "demo-password"             --- database password
};
```

If you use local database, you may need to [setup an ngrok channel](https://docs.mindsdb.com/sql/create/database#making-your-local-database-available-to-mindsdb) and use provided values for host and port.